crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -135,6 +137,27 @@ def finished?
       current_step == upgrade_steps_6_7.last
     end
 
+    def save_current_node(node_data = {})
```

Comment:
Ah, well. I'm still not sure what do we want to store there.

And there are some static data (name, IP), which are really nothing like "progress"